intro/index.html

```diff
@@ -64,7 +64,7 @@
 3. [**Warmup - expressions**](#expressions) - a quick tour of some fundamental expression building blocks
 4. [**Inside an attention head**](#attnhead) - an in-depth look at the structure, values and computation behavior of a couple of attention heads from GPT2 via[ NanoGPT](https://github.com/karpathy/nanoGPT)
 5. [**Parallelizing attention**](#attnpar) - visualizing attention head parallelization with examples from the recent[ Blockwise Parallel Transformer](https://arxiv.org/pdf/2305.19370.pdf) paper
-6. [**Sizes in an attention layer**](#attnsizes) - what do the MHA and FFA halves of an attention layer look like together, when we visualize a whole layer as a single structure? How does the picture change during autoregressive decoding?
+6. [**Sizes in an attention layer**](#attnsizes) - what do the MHA (Multi-Headed Attention) and FFA (Firefly Algorithm) halves of an attention layer look like together, when we visualize a whole layer as a single structure? How does the picture change during autoregressive decoding?
```

Comment:
Ah you've actually found a typo, this acronym should be `FFN` (as below). Would you mind making the change to "FFN (Feed Forward Network)" here?

Reply:
No problem, I just did :)